intake_xarray/raster.py

```diff
@@ -104,7 +104,7 @@ def _get_schema(self):
 
             ds2 = xr.Dataset({'raster': self._ds})
             metadata = {
-                'dims': dict(ds2.dims),
+                'dims': dict(ds2.sizes),
```

Comment:
https://docs.xarray.dev/en/stable/generated/xarray.Dataset.sizes.html says 
> This is an alias for Dataset.dims provided for the benefit of consistency with DataArray.sizes.

but if it stops the warning, that's OK with me.